frontend/src/home/Home.tsx

```diff
@@ -583,6 +614,10 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
     dispatch(addNewSchedule(name, newSchedule)),
   updateActiveSchedule: (updatedSchedule: ScheduleSlice) =>
     dispatch(updateActiveSchedule(updatedSchedule)),
+  logOut: () => {
+    dispatch(resetUserAction());
+    dispatch(resetScheduleAction());
```

Comment:
It's generally not great practice to call dispatch twice in one prop. What I would recommend is to separate this out into two props: `resetUserAction` and `resetScheduleAction`, and then call both of those in the `logout` function.